src/main/kotlin/com/neva/gradle/fork/gui/PropertyDialogField.kt

```diff
@@ -0,0 +1,60 @@
+package com.neva.gradle.fork.gui
+
+import com.neva.gradle.fork.config.properties.Property
+import com.neva.gradle.fork.config.properties.Validator
+import java.awt.Color
+import javax.swing.*
+
+class PropertyDialogField(private val property: Property, private val propField: JComponent, private val validationMessageLabel: JLabel, private val dialog: JDialog) {
+
+  init {
+    setPropertyValue()
+  }
+
+  val name: String
+    get() = property.name
+
+  val value: String
+    get() = property.value
+
+  fun validateAndDisplayErrors(): Boolean {
+    setPropertyValue()
```

Comment:
`set` indicates setter; please avoid setter like methods which are not setters actually

maybe better `assign`

Reply:
OK